### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Henry-Sarabia** [Henry-Sarabia/discovernow#54]
The landing page is missing appropriate aria labels and semantic markup that is used for accessibility tools. These should be implemented to improve accessibility.
